tests/zfs-tests/tests/functional/cli_root/zfs_set/property_alias_001_pos.ksh

```diff
@@ -22,13 +22,11 @@
 
 #
 # Copyright 2009 Sun Microsystems, Inc.  All rights reserved.
-# Use is subject to license terms.
-#
-
-#
 # Copyright (c) 2016, 2017 by Delphix. All rights reserved.
```

Comment:
proudly presented ***by*** ;) you know what I mean :D

Reply:
Yeah, Women fashion by Delphix... The name even sounds like a fashion brand for 50+ women.